gpt_json/tests/test_truncation.py

```diff
@@ -49,7 +49,7 @@ class TestSchema(BaseModel):
     gpt = GPTJSON[TestSchema](None)
 
     # this should fail because the max_prompt_tokens is too small
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError) as excinfo:
```

Comment:
Probably easier to use the regex pattern, no? `pytest.raises(Exp, match=r".* 123 .*")`?